<launch>
  <include file="$(find pr2_machine)/$(env ROBOT).machine" />
  <include file="$(find flirtlib_ros)/move_base_local.xml"/>
  <node ns="move_base_local" pkg="move_base" type="move_base" name="move_base_node" machine="c2">
    <remap from="odom" to="/base_odometry/odom" />
    <remap from="cmd_vel" to="/navigation/cmd_vel" />

    <!-- Load common configuration files -->
    <rosparam file="$(find flirtlib_ros)/launch/local_nav/move_base_params.yaml" command="load"/>
    <rosparam file="$(find pr2_navigation_config)/move_base/costmap_common_params.yaml"
              command="load" ns="global_costmap" />
    <rosparam file="$(find pr2_navigation_config)/move_base/costmap_common_params.yaml"
              command="load" ns="local_costmap" />
    <rosparam file="$(find flirtlib_ros)/launch/local_nav/base_local_planner_params.yaml"
              command="load" />
    <rosparam file="$(find flirtlib_ros)/launch/local_nav/local_costmap_params.yaml"
              command="load" ns="local_costmap"/>
    <rosparam file="$(find flirtlib_ros)/launch/local_nav/global_costmap_params.yaml"
              command="load" ns="global_costmap"/>
  </node>
</launch>
